project-reports/2024/2024-Q3-Hyperledger-Cello.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: default
+title: 2024 Q3 Hyperledger Cello
+parent: 2024
+grand_parent: Project Updates
+has_children: false
+has_toc: false
+---
+
+# Project Health
+
+
+The Hyperledger Cello project remains in good health, marked by the recent update. This update includes key dependency upgrades and bug fixes.
+
+* Upgraded Cello dashboard's Node.js version.
+* Upgraded Cello Backend API Engine Dependencies: 
+  * djangorestframework: 3.11.2 --> 3.15.1;
+  * drf-yasg: 1.17.1 --> 1.21.7;
+  * packaging: 20.4 --> 21.0 
+* Kicked off Linux Foundation Mentorship with two interns:
+  * Haibo Yang
+  * Haoyu Li
+
+
+# Questions/Issues for the TOC
+
+There are currently no outstanding issues.
+
+# Releases
+
+* v1.1 is planned to be released in 2024
+* v1.0.0-GA was released in October 2023.
+* v1.0.0-RC2 was released in September 2023.
+* v1.0.0-RC1 was released in June 2023.
+* v1.0.0-beta1 was completed in November 2022.
+
+# Overall Activity in the Past Quarter
+
+The Cello team has focused on addressing issues in released versions, with significant progress on several key tasks:
+
+* Fixed and tested the issue related to uploading Chaincode.
+* Onboarded two interns to contribute to the Cello project.
+* Proposed a new server-based component to abstract business logic for creating Fabric networks.
+* Discussed and planned the key features for the next version.
```

Comment:
Do you have a link to the roadmap for the project?

Reply:
Thanks for the feedback, we haven't had the roadmap yet but started a thread to discuss this https://github.com/hyperledger/cello/issues/634